foxy/distribution.yaml

```diff
@@ -1217,6 +1217,17 @@ repositories:
       url: https://github.com/swri-robotics/marti_messages.git
       version: dashing-devel
     status: developed
+  mavlink:
+    release:
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/mavlink/mavlink-gbp-release.git
+      version: 2020.8.8-1
+    source:
+      type: git
+      url: https://github.com/mavlink/mavlink-gbp-release.git
```

Comment:
The source entry should point to the source repository (not the release repository). Please update the url to https://github.com/mavlink/mavlink.git and the version to the branch or tag used for the Foxy release.

Reply:
The source repo do not have `package.xml`, previously that leads to errors, so for all distributions patched branch were used.

Reply:
Ah okay, I didn't realize. Sounds good.